### PR TITLE
Lesson-4-1

### PR DESCRIPTION
**Pull request by Armanchi**

### CODE REVIEW

File: src/Components/TodoContainer.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        <p>Loading...</p>
+          ) : (
+        <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
+  )}
+  </div>
+      </>
+    )
+}
+
+TodoContainer.propTypes = {
```
Comment (kramduckner) [inline src/Components/TodoContainer.js]: I don't see these props being passed into TodoContainer (on line 9)

File: src/Components/InputWithLabel.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      name="title" 
+      value={todoTitle}
+      onChange={handleTitleChange}
+      ref={inputRef}
+    />
+    </>
+  )
+}
+
+InputWithLabel.propTypes = {
```
Comment (kramduckner) [inline src/Components/InputWithLabel.js:24]: todoTitle, handleTitleChange, children are the only props being defined for InputWithLabel (see line 4 in this file)

**Review by kramduckner — CHANGES_REQUESTED** [review]
Nice work! It's super close and I'm not seeing any prop type warnings! Just a couple comments about ensuring the props are being used in the components

File: src/Components/TodoContainer.js (hunk 1 above)
Comment (Armanchi) [reply src/Components/TodoContainer.js]: I just pushed the changes!

Comment (kramduckner) [reply src/Components/TodoContainer.js]: tytyty

**Review by kramduckner — APPROVED** [review]
Nicely Done!

**Pull request merged**